framework/player/init/groupTracker.sqf

```diff
@@ -0,0 +1,128 @@
+// GROUP MARKERS ALTERNATE /////////////////////////////////////////////////////////////////////////
+/*
+	- Creates group markers following around the player groups. When clicked while map is open,
+	  it will list current members of the group.
+*/
+// INIT ///////////////////////////////////////////////////////////////////////////////////////////
+//GENERAL GROUP ICON SETTINGS
+if !(hasInterface) exitWith {};
+setGroupIconsVisible [true,false];
+setGroupIconsSelectable true;
+var_iconColor = [0,0,0,1];
+if (toUpper var_markerSide isEqualTo "COLORWEST") then {var_iconColor = [0,0.3,0.6,1]};
+if (toUpper var_markerSide isEqualTo "COLOREAST") then {var_iconColor = [0.5,0,0,1]};
+if (toUpper var_markerSide isEqualTo "COLORGUER") then {var_iconColor = [0,0.5,0,1]};
+
+
+// FUNCTION CREATE THE GROUP ICON /////////////////////////////////////////////////////////////////
+private _lmf_fnc_createIcon = {
+	params [["_grp",grpNull],["_type","b_inf"],["_txt",""],["_scale",1]];
+
+	clearGroupIcons _grp;
+	_grp addGroupIcon [_type,[0,0]];
+	_grp setgroupIconParams [var_iconColor,_txt,_scale,true];
+};
+
+
+// WHICH GROUPS GET TRACKED ///////////////////////////////////////////////////////////////////////
+//FOX PLATOON
+if !(isNil "Grp_FOX_6") then {[Grp_FOX_6,"b_hq","FOX6",1] call _lmf_fnc_createIcon};
```

Comment:
You can remove the 1 as last parameter, the private function already does that, unless you want some other size, what is the point.